external_confs/hydra/sweeper/nicopp_vqgan.yaml

```
@@ -0,0 +1,46 @@
+#
+# usage: hydra/sweeper=nicopp_vqgan
+#
+---
+defaults:
+  - optuna
+  - override sampler: qmc
+
+sampler:
+    seed: 0
+    qmc_type: halton
+    scramble: true
+
+direction:
+    - maximize
+
+storage: null
+study_name: supmatch
+n_trials: 40
+n_jobs: 4
+max_failure_rate: 0.0
+search_space: null
+custom_search_space: null
+
+params:
+    # alg.ga_steps: choice(3)
+    alg.num_disc_updates: range(3, 6)
+    # alg.enc_loss_w: tag(log, interval(1.e-2, 1.0))
+    alg.disc_loss_w: tag(log, interval(1.e-3, 3.e-1))
+    alg.prior_loss_w: choice(null, 1.e-2, 1.e-3)
+    alg.pred_y_loss_w: choice(0.0, 1.0)
+    alg.twoway_disc_loss: choice(true, false)
+    alg.lr: tag(log, interval(1.e-6, 1.e-4))
+    alg.max_grad_norm: choice(null, 5, 10)
```

apparently you can't mix value types

```suggestion
    alg.prior_loss_w: choice(0.0, 1.e-2, 1.e-3)
    alg.pred_y_loss_w: choice(0.0, 1.0)
    alg.twoway_disc_loss: choice(true, false)
    alg.lr: tag(log, interval(1.e-6, 1.e-4))
    alg.max_grad_norm: choice(5, 10, 10000)
```